server/sockets/socket-server.js

```diff
@@ -0,0 +1,74 @@
+const {io} = require('./../index')
+
+// on connecting with  client socket
+io.on('connection', (socket) => {
+  console.log('connected')
+  // join event listner
+  socket.once('join', function (roomname) {
+    // checks for maximum users and if game started?
+    if ((io.nsps['/'].adapter.rooms[roomname] && io.nsps['/'].adapter.rooms[roomname].length > 11) || (io.nsps['/'].adapter.rooms[roomname] && io.nsps['/'].adapter.rooms[roomname].allowed != undefined)) {
```

Comment:
Check if you can make this if condition smaller than it is. Using variables maybe?